dokku

```diff
@@ -122,9 +122,9 @@ case "$1" in
 
   cleanup)
     # delete all non-running container
-    docker ps -a | grep 'Exit' | awk '{print $1}' | xargs docker rm &> /dev/null &
+    docker rm $(docker ps -a -f "status=exited" -q) &> /dev/null &
     # delete unused images
-    docker images | grep '<none>' | awk '{print $3}' | xargs docker rmi &> /dev/null &
+    docker rmi $(sudo docker images -f "dangling=true" -q) &> /dev/null &
```

Comment:
Fixed the sudo thing, was just cruft leftover.

> Be careful, when there is no image to delete, you will have an error

The same would happen with the old version too. BTW, your 2-liners look fine to me.